packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -858,3 +876,16 @@ export enum StreamEncryption {
    */
   MANAGED = 'MANAGED'
 }
+
+/**
+ * Specifies the capacity mode to apply to this stream.
+ */
+export enum StreamModeDetails {
+  /**
+   * Specify the provisioned capacity mode.
```

Comment:
It would be extremely helpful for users if there was some information here about what each of these values _mean_. That would help users choose what's the right value for their use case without having to search for external documentation.

Reply:
Yes. It would also be helpful if CloudFormation bothered to document this properly. Since CloudFormation's docs are authoritative, I'd be nervous about too much interpreted detail in case we diverge from the truth. Nonetheless, we can try to do a better job here.

Reply:
I've attempted to make this clearer without going into too much detail.